tests/python-pytest/onnx/test_operators.py

```diff
@@ -452,3 +452,15 @@ def test_onnx_export_reshape_like(tmp_path, dtype):
     op_export_test('reshape_like3', M3, [x, y], tmp_path)
     M4 = def_model('reshape_like', lhs_begin=0, lhs_end=None, rhs_begin=1, rhs_end=None)
     op_export_test('reshape_like4', M4, [x, y], tmp_path)
+
+
+@pytest.mark.parametrize('dtype', ['int32', 'int64', 'float16', 'float32', 'float64'])
+def test_onnx_export_gather_nd(tmp_path, dtype):
+    x1 = mx.nd.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]], dtype=dtype)
```

Comment:
Shall we use high dimensional inputs and randomized inputs? something like
```
x = mx.random.uniform(0, 100, (7, 7, 7)).astype(dtype)
y = mx.random.uniform(0, 7, shape ).astype(dtype)
```

Reply:
Fixed